internal/plugins/ansible/v1/scaffolds/api.go

```diff
@@ -88,8 +87,7 @@ func (s *apiScaffolder) scaffold() error {
 
 	// Check that the provided group can be added to the project
 	if !s.config.MultiGroup && len(s.config.Resources) != 0 && !s.config.HasGroup(resourceOptions.Group) {
-		return fmt.Errorf("multiple groups are not allowed by default, to enable multi-group visit %s",
-			"kubebuilder.io/migration/multi-group.html")
+		return errors.New("support for multiple groups is disabled; to enable, set `multigroup: true` in the PROJECT file")
```

Comment:
For consistency with the Go error:
```suggestion
		return errors.New("multiple groups are not allowed by default, to enable multi-group set 'multigroup: true' in your PROJECT file")
```